src/main/java/org/commonjava/storage/pathmapped/core/PathMappedFileManager.java

```diff
@@ -215,4 +215,9 @@ public void close() throws IOException
             ( (Closeable) pathDB ).close();
         }
     }
+
+    public void clearAll()
```

Comment:
I'm thinking if we need to expose this type of operation in high level interface as it is too dangerous. Maybe we just need to hold it in lower level APIs like CassandraPathDB or JPAPathDB (even not to leave it in PathDB interface), and if test want to use it, they should directly call from the PathDB impl itself to do double check.

Reply:
Agree. I will close this. I have got another way to clean up in Indy ftests.